lib/index.js

```diff
@@ -1,5 +1,9 @@
 /* eslint no-process-env: 0, no-console: 0, max-statements: 0 */
 'use strict';
+process.on('unhandledRejection', function(e) {
+  // no swallowed Promise errors
+  throw e;
```

Comment:
I see what you mean. I wasn't aware people were using "swallows" for scenarios like this, makes sense:

```es6
somePromise.then(() => {
  someThingThatThrows();
}); // no catch!
```

I wouldn't say that the Promise is swallowing the error, only that the developer is ignoring the error.

:+1: if this can provide visibility to ignored errors during the running of tests, but I think it'd be better if it could be turned on with an option in case it causes issues.